platform/android/java/editor/src/main/java/org/godotengine/editor/BaseGodotEditor.kt

```diff
@@ -190,6 +190,7 @@ abstract class BaseGodotEditor : GodotActivity(), GameMenuFragment.GameMenuListe
 			// The RECORD_AUDIO permission is requested when the "audio/driver/enable_input" project
 			// setting is enabled.
 			Manifest.permission.RECORD_AUDIO,
+			Manifest.permission.CAMERA,
```

Comment:
Can you add a comment to describe when the `CAMERA` permission is being used for the editor.

Reply:
Comment added on `CAMERA` permission.